### PR TITLE
Alerts for NOR bot

### PR DESCRIPTION
**Pull request by hweawer**

### CODE REVIEW

File: alerts/alerts.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+groups:
+  - name: Distribution bot. Unexpected exceptions.
+    rules:
+      - alert: DistributionBotUnexpectedErrors
+        expr: increase(distribution_bot_unexpected_exceptions{service="nor-reward-distribution-bot"}[1h]) > 0
+        labels:
+          severity: critical
```
Comment (F4ever) [inline alerts/alerts.yml:7]: Let's make it `medium`

File: alerts/alerts.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          app_team: tooling
+        annotations:
+          summary: "Distribution bot got an unexpected exception."
+          description: "Unexpected exception has happened."
```
Comment (F4ever) [inline alerts/alerts.yml:11]: Let's add here label from metric.

Comment (F4ever) [reply alerts/alerts.yml:11]: by the way, same alert can be added to depositor

File: alerts/alerts.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  - name: Distribution bot. Account balance.
+    rules:
+      - alert: DistributionBotLowAccountBalance
+        expr: distribution_bot_account_balance{service="nor-reward-distribution-bot"}/10^18 < 0.5
+        for: 15m
```
Comment (F4ever) [inline alerts/alerts.yml:16]: remove for

File: alerts/alerts.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        labels:
+          severity: warning
```
Comment (F4ever) [inline alerts/alerts.yml:18]: make it low

File: alerts/alerts.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+          app_team: tooling
+        annotations:
+          summary: "Low account balance for distribution bot {{ $value }} ETH."
+  - name: Distribution bot. Distributions.
+    rules:
+      - alert: DistributionBotLowDistributions
+        expr: max_over_time(distribution_bot_rewards_distribution_status[15m]) != 2
+        for: 15m
+        labels:
+          severity: critical
```
Comment (F4ever) [inline alerts/alerts.yml:28]: Make it medium.

File: alerts/alerts.yml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        annotations:
+          summary: "Distributor bot didn't distribute anything in 15 minutes."
+          description: "Distributor bot stuck distributing."
+  - name: Distribution bot. Head is stale.
+    rules:
+      - alert: DistributionBotStaleHeadBlock
+        expr: changes(distribution_bot_latest_block_number[15m]) == 0
+        for: 15m
+        labels:
+          severity: critical
```
Comment (F4ever) [inline alerts/alerts.yml:39]: also high

File: alerts/alerts.yml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+          app_team: tooling
+        annotations:
+          summary: "Distributor bot didn't distribute anything in 15 minutes."
+          description: "Distributor bot stuck distributing."
+  - name: Distribution bot. Head is stale.
+    rules:
+      - alert: DistributionBotStaleHeadBlock
+        expr: changes(distribution_bot_latest_block_number[15m]) == 0
+        for: 15m
```
Comment (F4ever) [inline alerts/alerts.yml:37]: We can remove for here

File: alerts/alerts.yml [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+        expr: distribution_bot_account_balance{service="nor-reward-distribution-bot"}/10^18 < 0.5
+        for: 15m
+        labels:
+          severity: warning
+          app_team: tooling
+        annotations:
+          summary: "Low account balance for distribution bot {{ $value }} ETH."
+  - name: Distribution bot. Distributions.
+    rules:
+      - alert: DistributionBotLowDistributions
```
Comment (F4ever) [inline alerts/alerts.yml:24]:
```suggestion
      - alert: DistributionBotNoDistributions
```

File: alerts/alerts.yml [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+        expr: max_over_time(distribution_bot_rewards_distribution_status[15m]) != 2
+        for: 15m
+        labels:
+          severity: critical
+          app_team: tooling
+        annotations:
+          summary: "Distributor bot didn't distribute anything in 15 minutes."
+          description: "Distributor bot stuck distributing."
```
Comment (F4ever) [inline alerts/alerts.yml:32]: Add `address` label of Staking Module to description

File: alerts/alerts.yml (hunk 1 above)
Comment (hweawer) [reply alerts/alerts.yml]: Changed to `major`, we don't use `medium` severity

File: alerts/alerts.yml (hunk 2 above)
Comment (hweawer) [reply alerts/alerts.yml]: Done

File: alerts/alerts.yml (hunk 4 above)
Comment (hweawer) [reply alerts/alerts.yml]: Done

File: alerts/alerts.yml (hunk 5 above)
Comment (hweawer) [reply alerts/alerts.yml]: Done

File: alerts/alerts.yml (hunk 6 above)
Comment (hweawer) [reply alerts/alerts.yml]: Done

File: alerts/alerts.yml (hunk 7 above)
Comment (hweawer) [reply alerts/alerts.yml]: Done

File: alerts/alerts.yml (hunk 8 above)
Comment (hweawer) [reply alerts/alerts.yml]: Renamed

File: alerts/alerts.yml (hunk 9 above)
Comment (hweawer) [reply alerts/alerts.yml]: Done

File: README.md [hunk 10]
```diff
@@ -96,6 +96,8 @@ Alerts list:
 | nor_distributor_unexpected_errors | Unexpected errors. Check logs for details |
```
Comment (F4ever) [inline README.md:96]: Last thing - rename alerts in table to names used in alerts.yml

Comment (hweawer) [reply README.md]: Done

**Review by F4ever — APPROVED** [review]

**Pull request merged**